README.md

```diff
@@ -1 +1,189 @@
-# frontend-capstone
\ No newline at end of file
+<div align="center">
+  <h3><b>Property Rental App</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About the Project](#-about-the-project)
+  - [🛠 Built With](#-built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [💻 Getting Started](#-getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [🔭 Future Features](#-future-features)
+  - [🤝 Contributing](#-contributing)
+  - [⭐️ Show your Support](#-show-your-support)
+  - [🙏 Acknowledgments](#-acknowledgments)
+  - [📝 License](#-license)
+
+# 📖 About the Project <a name="about-project"></a>
+
+**Property Rental App** is a web application designed to facilitate the booking of appointments to view and rent properties. The project follows a provided design and layout but adapts the content to offer a platform for reserving appointments with properties for rent.
+
+Select a theme for your website - is it going to be a website for booking doctor appointments, booking online classes, or something else?
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <summary>CSS</summary>
+  <summary>JavaScript</summary>
+  <summary>Ruby on Rails</summary>
+  <summary>React</summary>
+  <summary>Redux</summary>
+  <summary>Postgres</summary>
+  <summary>Linters</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **Responsive design**
+- **User authentication with username and password**
+- **User can view list of properties for rent**
+- **User can view details of a specific property**
+- **User can reserve an appointment to view a property**
+- **User can view their reservations**
+- **User can add and delete properties**
+
+<!-- LIVE DEMO -->
+## [Live Demo]()
+
+A live demo link would be up soon
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project, you need:
+- To have the teck stack mentioned [above](#tech-stack);
+- To have the following repositories and follow the instructions in the respective README docs:
+    - [Front end](https://github.com/skabeo/frontend-capstone);
+    - [Back end](https://github.com/skabeo/backend_capstone)
+
+### Setup
+Clone this repository to your desired folder:
+
+```sh
+git clone https://github.com/skabeo/frontend-capstone.git
+cd frontend-capstone
+```
+
+## Install frontend dependencies
+npm install
+
+## Add your personal keys
+Navigate to ```src/api/sessionAPI.js``` and add your personal keys variables from the rails app as follow:
+```sh
+- const CLIENT_ID = 'your client id';
+- const CLIENT_SECRET = 'your client secret';
+```
+
+## Start React development server
+npm start
+
+## Jest
+> To run testing
+```
+npm test
+```
```

Comment:
- [ ] I got 2 frontend tests failing on my end.  Kindly fix those in order to secure your app's integrity. 👍 

![image](https://github.com/skabeo/frontend-capstone/assets/57421823/8a47a045-25f6-4978-add1-9201ea4e86a2)